src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -14,4 +14,16 @@
     public static final Prefix PREFIX_TAG = new Prefix("t/");
     public static final Prefix PREFIX_WEB_LINK = new Prefix("w/");
 
```

Comment:
It may have been possible to utilise the Prefix Class for this purpose. It could reduce the number of lines of code added in this case. Just as a food for thought.